doc_src/cmds/path.rst

```diff
@@ -0,0 +1,402 @@
+.. _cmd-path:
+
+path - manipulate and check paths
+=================================
+
+Synopsis
+--------
+
+::
+
+    path basename GENERAL_OPTIONS [PATH...]
+    path dirname GENERAL_OPTIONS  [PATH...]
+    path extension GENERAL_OPTIONS [PATH...]
+    path filter GENERAL_OPTIONS [(-v | --invert)] \
+        [-d] [-f] [-l] [-r] [-w] [-x] \
+        [(-t | --type) TYPE] [(-p | --perm) PERMISSION] [PATH...]
+    path is GENERAL_OPTIONS [(-v | --invert)] [(-t | --type) TYPE] \
+        [-d] [-f] [-l] [-r] [-w] [-x] \
+        [(-p | --perm) PERMISSION] [PATH...]
```

Comment:
no need for backslashes and "..." should have a leading space.  Also `[(-v | --invert)]` should be `[-v | --invert]` and options should probably be aligned in the source like in string.rst.